### PR TITLE
version notes

### PR DESCRIPTION
**Pull request by kbrock**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+a nice looking [Changelog](http://keepachangelog.com).
+
+## Version [HEAD] <sub><sup>now</sub></sup>
+
+## Version [0.3.2] <sub><sup>2022-02-02</sub></sup>
+
+- fixed keyword arguments for ruby 3.0 support #21
+- loosened active record support to include active record 6.1 #23
+
+[HEAD]: https://github.com/ManageIQ/activerecord-id_regions/compare/v4.1.0...HEAD
```
Comment (jrafanie) [inline CHANGELOG.md:13]: Should this be:

```
https://github.com/ManageIQ/activerecord-id_regions/compare/v0.3.2...HEAD
```

?

**Comment by Fryguy** [comment]
@kbrock Out of curiosity, where did you get this Changelog template?  It doesn't match the ones we have elsewhere, such as https://github.com/ManageIQ/more_core_extensions/blob/master/CHANGELOG.md?plain=1, nor does it match what is recommended at keepachangelog.com.

**Comment by kbrock** [comment]
Huh. must have over paired it down too much. will beef it back up. thanks

**Review by jrafanie — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## Version [Unreleased] <sub><sup>now</sub></sup>
+
+## Version [0.3.2] <sub><sup>2022-02-02</sub></sup>
+
+### Fixed
+
+- Loosened active record support to include active record 6.1 #23
+- Fixed keyword arguments for ruby 3.0 support #21
```
Comment (jrafanie) [inline CHANGELOG.md:13]: links here aren't automatic I guess...

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Change log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## Version [Unreleased] <sub><sup>now</sub></sup>
+
+## Version [0.3.2] <sub><sup>2022-02-02</sub></sup>
```
Comment (Fryguy) [inline CHANGELOG.md:8]:
```suggestion
## [Unreleased]

## [0.3.2] - 2022-02-02
```

File: CHANGELOG.md (hunk 2 above)
Comment (Fryguy) [reply CHANGELOG.md:13]: No, they need to be markdowned out

Comment (Fryguy) [inline CHANGELOG.md:13]:
```suggestion
- Loosened active record support to include active record 6.1 [[#23](https://github.com/ManageIQ/activerecord-id_regions/pull/23)]
- Fixed keyword arguments for ruby 3.0 support [[#21](https://github.com/ManageIQ/activerecord-id_regions/pull/21)]
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Change log
```
Comment (Fryguy) [inline CHANGELOG.md:1]: Super minor but consistent with other ones

```suggestion
# Change Log
```

File: CHANGELOG.md (hunk 2 above)
Comment (kbrock) [reply CHANGELOG.md]: No one else adds links to PR numbers and it adds a lot of noise for the file.
Thinking about just dropping all of this

**Comment by kbrock** [comment]
Do we want to move over to using actions and appraisals? like https://github.com/ManageIQ/activerecord-virtual_attributes/blob/master/.github/workflows/ci.yaml

**Comment by Fryguy** [comment]
Appraisals, no - cc @jrafanie and the work in the other gems can apeak more to that.
Actions, yes, but I am going to do that in a follow up - probably shouldn't be in this PR anyway, and it doesn't affect a release nor even the gem itself.

**Comment by kbrock** [comment]
we ready to merge this and release this gem?

**Comment by jrafanie** [comment]
> we ready to merge this and release this gem?

yup! YOLO

**Pull request merged**